__tests__/App-test.js

```diff
@@ -34,6 +34,8 @@ import {XamlExamplePage} from '../src/examples/XamlExamplePage';
 //import {TrackPlayerExamplePage} from '../src/examples/TrackPlayerExamplePage';
 import {WindowsHelloExamplePage} from '../src/examples/WindowsHelloExamplePage';
 import {ExpanderExamplePage} from '../src/examples/ExpanderExamplePage';
+// Unable to import: {export * from './types'; SyntaxError: Unexpected token 'export'
+//import {RadioButtonsExamplePage} from '../src/examples/RadioButtonsExamplePage';
```

Comment:
Do you want to leave this test in here? If so, let's file an issue and add it here to mark fixing this later.

Reply:
Yep, let me file an issue to track enabling this test. It's probably possible with a test config setting, but I felt out of my depth on that.